ods_ci/tests/Resources/Common.robot

```diff
@@ -305,3 +305,38 @@ Extract URLs From Text
     [Arguments]    ${text}
     ${urls}=     Get Regexp Matches   ${text}   (?:(?:(?:ftp|http)[s]*:\/\/|www\.)[^\.]+\.[^ \n]+)
     RETURN    ${urls}
+
+Run And Watch Command
+  [Documentation]    Run any shell command (including args) with optional:
+  ...    Timeout in minutes: 10 by default.
+  ...    Expected text: Verify the text exists in command output.
+  [Arguments]    ${command}    ${timeout_min}=10    ${expected_text}=${NONE}
+  Log    Watching command output: ${command}   console=True
+  @{args} =    Split String    ${command}
+  ${process_log} =    Set Variable    ${OUTPUT DIR}/${args}[0].log
+  Create File    ${process_log}
+  Create File    ${process_log}.old
+  ${process} =    Start Process    ${command}    shell=True    stdout=${process_log}    stderr=STDOUT    # robocop: disable
+  Log    Shell process started in the background   console=True
+  ${timeout_result} =    Wait Until Keyword Succeeds    ${timeout_min} min    10 s
+  ...    Read Command Log    ${process}    ${process_log}
+  ${proc_result} =	    Wait For Process    ${process}    timeout=3 secs
```

Comment:
Since the previous step was "Wait Until Keyword Succeeds" with a timeout of 10 minutes by default, I excpect the command to end before 10 minutes passed. 
Then I also give it a "gesture" of 3 more seconds to complete, as it has already passed the allowed timeout, or ended before the timeout. But it will return the process results, and that's what matters. to evaluate.